project/prepare-release.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+set -e
+
+# NOTES:
+#     To delete a tag use: git push origin :v1.0.0
+
+if [[ $1 != "patch" && $1 != "minor" && $1 != "major" ]] ; then
+  echo "    Bad input: \"$1\". Expected an update type of \"patch\", \"minor\" or \"major\"";
+  exit 1;
+fi
+
+echo ""
+echo ""
+echo "Publishing Propel $PACKAGE_VERSION"
+echo ""
+echo ""
+echo "Building Library"
+echo ""
+gulp default --throw-error
+
+echo ""
+echo ""
+echo "Building Docs"
+echo ""
+esdoc -c esdoc.json
+
+echo ""
+echo ""
+echo "Update NPM Version"
+echo ""
+npm --no-git-tag-version version $1
+
+PACKAGE_VERSION=$(cat package.json \
```

Comment:
Great idea - but then using your idea, I realised I could do this:

    PACKAGE_VERSION=$(npm --no-git-tag-version version $1)